cadence-contracts/PDS.cdc

```diff
@@ -13,32 +12,80 @@ pub contract PDS{
     pub let distCreatorPrivPath: PrivatePath
     pub let distManagerStoragePath: StoragePath
 
+    pub struct Collectible: IPackNFT.Collectible {
+        pub let address: Address
+        pub let contractName: String
+        pub let id: UInt64
+
+        // returning in string so that it is more readable and anyone can check the hash
+        pub fun hashString(): String {
+            // address string is 16 characters long with 0x as prefix (for 8 bytes in hex)
+            // example: ,f3fcd2c1a78f5ee.ExampleNFT.12
+            let c = ",A."
```

Comment:
Do we need to add the "," here? Could we do it in the `_verify` method of a Pack?